packages/server/src/migrations/schema/v53.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Generated by @medplum/generator
+ * Do not edit manually.
+ */
+
+import { getResourceTypes } from '@medplum/core';
+import { PoolClient } from 'pg';
+
+export async function run(client: PoolClient): Promise<void> {
+  for (const resourceType of getResourceTypes()) {
```

Comment:
Hm.... We may need to be careful with this.  The return value of `getResourceTypes` could change in the future, which would make this migration non-deterministic.

For all other migrations, we've intentionally duplicated resource types to ensure that migrations do the same thing.

Happy to discuss offline.  Depending on our assumptions about the new migration tooling, this may or not be a big deal.